requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django==2.1.10
+django-model-utils==3.2.0
+openwisp-utils==0.2.2
```

Comment:
let's move these to `install_requires` in `setup.py`, we stopped using this way of doing things